.github/workflows/tests.yml

```diff
@@ -24,9 +24,9 @@ jobs:
     - name: Build
       run: cargo build --verbose
     - name: Meilisearch (latest version) setup with Docker
-      run: docker run -d -p 7700:7700 getmeili/meilisearch:latest ./meilisearch --no-analytics=true --master-key=masterKey
+      run: docker run -d -p 7700:7700 getmeili/meilisearch:latest ./meilisearch --no-analytics --master-key=masterKey
```

Comment:
```suggestion
      run: docker run -d -p 7700:7700 getmeili/meilisearch:latest ./meilisearch --no-analytics=true --master-key=masterKey
```

Reply:
Not yet, will be for v0.26.0, not v0.25.0 :)

Reply:
I am not sure that I understand that, the previous version of meilisearch works with this change, and the newer version (v0.26) will only work with this change. This patch makes the command line compatible with the future version of Meilisearch and doesn't break the v0.25 one.

Can't we accept this change?

Reply:
Hum from what I see it has been merged some times ago already; https://github.com/meilisearch/meilisearch/pull/1984
And it works for me on meilisearch v0.25.2